<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frmLoginFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>851</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>146</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Введіть пароль для авторизації</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPassword">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btLogin">
         <property name="text">
          <string>Ввійти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btCancel">
         <property name="text">
          <string>Вихід</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>145</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frmMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btApps">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46,46,46);
border: 0px solid gray;
border-radius: 4px;
background: url(:/Img/Images/Apps.jpg) top center no-repeat;
padding-top: 32px;</string>
     </property>
     <property name="text">
      <string>



Програми</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btWeb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46,46,46);
border: 0px solid gray;
border-radius: 4px;
background: url(:/Img/Images/Web.jpg) top center no-repeat;
padding-top: 20px;</string>
     </property>
     <property name="text">
      <string>




Веб сайти</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btSettings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46,46,46);
border: 0px solid gray;
border-radius: 4px;
background: url(:/Img/Images/Settings.jpg) top center no-repeat;
padding-top: 24px;</string>
     </property>
     <property name="text">
      <string>

Налаштування</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btExit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46,46,46);
border: 0px solid gray;
border-radius: 4px;
background: url(:/Img/Images/Exit.jpg) top center no-repeat;
padding-top: 32px;</string>
     </property>
     <property name="text">
      <string>



Вихід</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btLock">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46,46,46);
border: 0px solid gray;
border-radius: 4px;
background: url(:/Img/Images/Lock.jpg) top center no-repeat;
padding-top: 32px;</string>
     </property>
     <property name="text">
      <string>



Заблокувати</string>
     </property>
    </widget>
   </widget>
   <widget class="AppSettings" name="AppWidget" native="true">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>-1</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="WebSettings" name="WebWidget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="SettingsWidget" name="StWidget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <zorder>WebWidget</zorder>
   <zorder>frmMenu</zorder>
   <zorder>frmLoginFrame</zorder>
   <zorder>AppWidget</zorder>
   <zorder>StWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AppSettings</class>
   <extends>QWidget</extends>
   <header location="global">AppSettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WebSettings</class>
   <extends>QWidget</extends>
   <header location="global">WebSettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">SettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
